### PR TITLE
Add categories from database to the table.

### PR DESCRIPTION
**Pull request by imila254**
Possibility to delete value from the table.

### CODE REVIEW

File: Desktop/MyBudgetWindow.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            if (budgetAndCategoriesView.CurrentRow != null)
+            {
+               // DataGridViewRow dgwRow = new DataGridViewRow();
+                /* Category newCategory = new Category
+                 {
+                     Id = 1,
+                     Title = dgwRow.Cells["txtCategory"].Value.ToString(),
+                     Description = dgwRow.Cells["txtBudgeted"].Value.ToString()
+
+                 };*/
```
Comment (barkas90) [inline Desktop/MyBudgetWindow.cs:62]: What is the point of this If function?

Comment (imila254) [reply Desktop/MyBudgetWindow.cs:62]: It's still in progress. This method will take the information from the table and update the database.

File: Desktop/MyBudgetWindow.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        public void UpdateCategoriesList()
+        {
+            CategoryList = db.GetCategories();
+            PopulateCategoryListView();
+        }
+
+        private void PopulateCategoryListView()
+        {
+            PopulateCategoryListView(CategoryList);
+        }
```
Comment (sk057) [inline Desktop/MyBudgetWindow.cs:37]: What is the point of this method?

File: Desktop/MyBudgetWindow.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+        private void PopulateCategoryListView(IEnumerable<Category> CategoryList)
+        {
+            this.budgetAndCategoriesView.Rows.Add("1", "Shopping", "30", "30", "0");
```
[the remark this answers is not in the document]
Comment (imila254) [reply Desktop/MyBudgetWindow.cs]: Just test line

File: Desktop/MyBudgetWindow.cs (hunk 2 above)
Comment (imila254) [reply Desktop/MyBudgetWindow.cs:37]: To get data from the database and display it in a window.

Comment (sk057) [reply Desktop/MyBudgetWindow.cs:37]: But why do you not just call the method in line 39 from line 31?

**Review by sk057 — APPROVED** [review]

**Pull request merged**